CONTRIBUTING.md

```diff
@@ -48,17 +53,69 @@ If the pull request does not have a clear purpose and benefits, it will not be a
+
+    * A release note does not need to be written for changes to the code that users will not see. Here are some examples. If you add a
+comment, you do not need to write a release note describing the comment you added. If you rewrite some code to make it read more
+clearly and if there is no change in functionality or performance, then you do not need to write a release note. If you change the
+process by which user software is made, you may not need to write a release note since the change was not made to the code.
+
+    * Users. We have different kinds of users. A release note may be written to be helpful to
+application developers and not system administrators. Users who may find the RELEASE.txt file helpful include the following:
+application developers, library developers, and system administrators.
+
+
+# Release Note <A NAME="releasenote"></A>
+
+* **Entry Syntax**
+The release note entry syntax is shown below.
+
+```
+    - Title/Problem
+
+      Problem/Solution
+
+      Signature
+```
+
+* **Entry Elements** - The elements of the entry - title, problem, solution, and signature - are described in more detail in the table
+below. Descriptions of the problem and the solution should be clear without any ambiguities and should be short without losing clarity or specifics.
+
+    * **Title** - The title or tag should identify one or more categories that will help readers decide if the entry is something they need to study. Can be combined with the `Problem` element
+    * **Problem** - Describe the problem and how users might see the problem in a paragraph.
+You might also consider the following as you describe the problem:
+        * Under what specific conditions does this issue arise?
+        * Under what specific conditions are we sure this issue will not arise?
+        * For a performance issue, instead of saying something is a performance issue, describe what the performance impact of issue is?
+    * **Solution** - Describe the solution in another paragraph.
+You might also consider the following as you describe the solution:
+        * What was done to resolve the issue?
+        * What is the functional impact?
+        * Is there a workaround – a way for users design their software so as not to encounter the issue? If so, what is the workaround?
+        * For a performance fix, how has the performance improved? Links to published documentation would be good.
+    * **Signature** - Each entry must be signed with the initials of the author, the date in YYYY/MM/DD format, and the JIRA ticket number. The
+following is an example entry written by developer Xavier Zolo on April 16, 2014 about JIRA ticket HDFFV-5555: (XYZ - 2014/04/16, HDFFV-5555). The
+signature is enclosed in parentheses. JIRA or Github numbers should not be used in the description of the problem or the solution. They are like
+abbreviations that customers and external users will not be able to interpret.
```

Comment:
I don't think adding the developer is helpful. As an external person reading RELEASE.txt would you care?

Reply:
For internal use, just `git blame` the file.